.github/workflows/publish-test.yml

```diff
@@ -78,4 +67,10 @@ jobs:
           path: code-coverage-results.md
           number: ${{ env.TEST_PR_NUMBER }}
 
+      - name: Publish Event File
+        uses: actions/upload-artifact@v2
+        with:
+          name: event-file
+          path: ${{ github.event_path }}
```

Comment:
I'm leaving this in the workflow. The event file is very helpful when you need to tackle these types of problems.